halo2_proofs/src/plonk/circuit.rs

```diff
@@ -475,6 +546,13 @@ pub enum Expression<F> {
         /// Rotation of this query
         rotation: Rotation,
     },
+    /// This is a challenge
+    Challenge {
+        /// Challenge index
+        index: usize,
+        /// Challenge phase
+        phase: Phase,
+    },
```

Comment:
```suggestion
    Challenge(Challenge),
```

Reply:
Updated in [`902965c`](https://github.com/zcash/halo2/pull/593/commits/902965c385833ada61def929905c7431ff36e78b).